quam_builder/architecture/superconducting/custom_gates/flux_tunable_transmon_pair/two_qubit_gates.py

```diff
@@ -48,10 +57,18 @@ class CZGate(QubitPairMacro):
     phase_shift_target : float
          Default frame rotation (in units of 2π) applied to the target qubit after flux interaction
          if not overridden and |value| > 1e-6.
+    spectator_qubits: dict[str, Any]
```

Comment:
I would add a description about the intended usage of these new parameters above to guide the users and even show a usage example.